dhis2/dhis2-core
# feat: add fallback login page

## Summary
Adds a simple fallback login page in pure HTML that uses the default Spring login endpoint.

## Manually test
1. Go to ./login.html
2. Log in as usual with your username/password. (this does not support 2FA)
3. Observe you get logged in.

### JIRA:
[DHIS2-17682](https://dhis2.atlassian.net/browse/DHIS2-17682)

[DHIS2-17682]: https://dhis2.atlassian.net/browse/DHIS2-17682?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
